scripts/build

```diff
@@ -297,11 +303,28 @@ if [ ! -f $STAMP ]; then
         make_$TARGET
       else
         if [ "$TARGET" = "target" ]; then
-          echo "Executing (target): make $PKG_MAKE_OPTS_TARGET" | tr -s " "
-          make $PKG_MAKE_OPTS_TARGET
+          if [ "$PKG_IS_PYTHON" = "yes" ]; then
+            if [ "$PKG_PYTHON_CROSS_COMPILE" != "no" ]; then
+              PKG_PYTHON_OPTS_TARGET="--cross-compile"
+            fi
+            echo "Executing (target): python setup.py build $PKG_PYTHON_OPTS_TARGET" | tr -s " "
+            mkdir -p ".$TARGET_NAME"
+            cd ".$TARGET_NAME"
+            cp -r ../* .
+            export LDSHARED="$CC -shared"
```

Comment:
If I add an additional LDSHARED for instance to add "pthread" in the package.mk this will overwrite it.